hurricane/server/__init__.py

```diff
@@ -240,6 +240,29 @@ def check_db_and_migrations(webhook_url: str = None, loop: asyncio.unix_events.S
         raise e
 
 
+def check_db_and_migrations_and_apply(webhook_url: str = None, loop: asyncio.unix_events.SelectorEventLoop = None):
```

Comment:
The only difference to `check_db_and_migrations` are the two lines that apply the migrations. Let's refactor this with `check_db_and_migrations`, so it can be called with following signature:
```
def check_db_and_migrations(webhook_url: str = None, loop: asyncio.unix_events.SelectorEventLoop = None, apply_migrations: bool = False):
    [...]
```
And only run the migrate lines when `apply_migrations` is True. Can you please change that (and also change the function call in `serve.py`)?